src/style.css

```
@@ -1 +1,46 @@
-/* add styles here */
+* {
+  margin: 0;
+}
+
+.stars__blocks {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 > .stars__star:first-child,
.stars--2 > .stars__star:nth-child(-n + 2) {
  background-image: url(./images/star-active.svg);
}
```
combine it with comma

---

done